src/main/java/lotto/domain/Jackpot.java

```diff
@@ -0,0 +1,24 @@
+package lotto.domain;
+
+public enum Jackpot {
+    THREE(3, 5000),
+    FOUR(4, 50000),
+    FIVE(5, 1500000),
+    SIX(6, 2000000000);
```

Comment:
다음과 같이 언더스코어를 이용하여 가독성을 높일 수 있습니다 :)
```suggestion
    SIX(6, 2_000_000_000);
```

Reply:
오우! 언더스코어를 해도 숫자로 인식되는 건가요?
이건 몰랐는데 대단하네요.
엄청 좋습니다.